framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -2593,7 +2621,7 @@ public boolean isFromByteCode(Element element) {
      *
      * @return the annotation mirror for the requested annotation or null if not found
      */
-    protected AnnotationMirror getDeclAnnotation(Element elt,
+    private AnnotationMirror getDeclAnnotation(Element elt,
```

Comment:
I assume there are two public methods that use this method, one that passes true and another that passes false. This method contains the common implementation for both.
By making this method private you prevent a subclass from simply consistently adapting both public methods.
I would rather make the two public methods final and keep this one protected and clearly explain how a subclass can adapt this behavior.